config/routes.php

```diff
@@ -46,6 +46,11 @@
 Router::defaultRouteClass(DashedRoute::class);
 
 Router::scope('/', function (RouteBuilder $routes) {
+    Router::prefix('/api', function (RouteBuilder $routes) {
```

Comment:
Move this block of code outside scope('/'....

Reply:
done po